src/lib/constant_mappings.cpp

```diff
@@ -149,6 +149,12 @@ const boost::bimap<AggregateFunction, std::string> aggregate_function_to_string
         {AggregateFunction::CountDistinct, "COUNT DISTINCT"},
     });
 
+const boost::bimap<PartitionSchemaType, uint8_t> partition_schema_type_to_uint =
```

Comment:
if you need a mapping enum<->int, just explicitly assign a value to each enum member and cast http://en.cppreference.com/w/cpp/language/enum